numpy/core/defchararray.py

```diff
@@ -942,6 +942,14 @@ def isupper(a):
     See Also
     --------
     str.isupper
+
+     Examples
+    --------
+    >>> import numpy as np
+    >>> str="GHC"
```

Comment:
extreme nitpick, do add spaces around `=` in assignments (https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements)
```suggestion
    >>> str = "GHC"
```